interfaces/console/src/constants/index.tsx

```diff
@@ -465,12 +480,89 @@ export const SITE_KPIS = {
           max: 1,
         },
       },
+      {
+        unit: '',
+        show: true,
+        name: 'Solar switch port status',
+        id: 'solar_switch_port_status',
+        description: 'Solar controller switch port status (1 = up, 0 = down)',
+        tickInterval: 1,
+        min: 0,
+        max: 1,
+        threshold: {
+          min: 0,
+          normal: 1,
+          max: 1,
+        },
+      },
+      {
+        unit: '',
+        show: true,
+        name: 'Backhaul switch port status',
+        id: 'backhaul_switch_port_status',
+        description: 'Backhaul switch port status (1 = up, 0 = down)',
+        tickInterval: 1,
+        min: 0,
+        max: 1,
+        threshold: {
+          min: 0,
+          normal: 1,
+          max: 1,
+        },
+      },
```

Comment:
Please add format property too, can be `number/decimal`. And pass it to LineChart component